Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template/Resources/settings.json

```diff
@@ -0,0 +1,46 @@
+﻿{
```

Comment:
I believe it should be in Aquality.Appium.Mobile.Template.SpecFlow/settings.json